### PR TITLE
[WIP] Ticketpage

### PR DESCRIPTION
**Pull request by martijnjanssen**

### CODE REVIEW

File: cert.pem [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```
Comment (TimvdLippe) [inline cert.pem:1]: Lets add `*.pem` to the `.gitignore`

File: src/symposium-app/symposium-ajax.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+
+<dom-module id="symposium-ajax">
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-ajax.html:14]: This one looks oddly familiar 😉

File: src/symposium-app/symposium-app.html [hunk 3]
```diff
@@ -202,6 +207,10 @@
       properties: {
       },
 
+      ready: function() {
+        console.log(this.route);
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-app.html:211]: W00h00. "Left-over debug statement'

File: src/symposium-app/symposium-tickets/symposium-ordercompleted.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        Here the different ordering outcomes will be handled in the front-end.
+    </symposium-page-card>
+
+  </template>
+  <script>
+  (function() {
+    'use strict';
+
+    Polymer({
+      is: 'symposium-ordercompleted',
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-tickets/symposium-ordercompleted.html:22]: Lets make this `symposium-order-completed`

File: src/symposium-app/symposium-tickets/symposium-ordercompleted.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+<dom-module id="symposium-ordercompleted">
+  <template>
+    <style include="iron-flex iron-flex-alignment">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <symposium-page-card>
+        Here the different ordering outcomes will be handled in the front-end.
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-tickets/symposium-ordercompleted.html:13]: TODO

File: src/symposium-app/symposium-tickets/symposium-ordercompleted.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+    Polymer({
+      is: 'symposium-ordercompleted',
+
+      properties: {
+      },
+
+      ready: function() {
+          var order = window.location.href.split('order=')[1];
+          console.log(order);
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-tickets/symposium-ordercompleted.html:29]: Yup

File: src/symposium-app/symposium-tickets/symposium-tickets.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+      onTickets: function(e, request) {
+        if (request.succeeded) {
+          this.tickets = request.response.filter(function(t) {
+            return t.available;
+          }).sort(function(a, b) {
+            return a.price - b.price;
+          });
+        } else {
+          console.log('Handle error');
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-tickets/symposium-tickets.html:64]: TODO

File: src/symposium-app/symposium-tickets/symposium-order.html [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+      ],
+
+      tryCheckout: function() {
+        if (this.$.form.validate()) {
+          var tickets = this.order.map(function(t) {return t.key;});
+          this.$.ajaxOrder.body = {
+            productKeys: tickets,
+            name: this.$.form.getName(),
+            email: this.$.form.getEmail(),
+            returnUrl: 'https://localhost:8080/ordercompleted',
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-tickets/symposium-order.html:87]: TODO: change to correct url. You can do this with `window.location.protocol + '//' + window.location.hostname + (window.location.port && ':' + window.location.port) + '/ordercompleted'`

Comment (martijnjanssen) [reply src/symposium-app/symposium-tickets/symposium-order.html:87]: Ooh, that is a smart one! Saves a lot of errors later on changing to production and dev environment.

**Comment by martijnjanssen** [comment]
Closing this PR due to #52.

**Pull request closed without merging**